web/src/lib/types.ts

```diff
@@ -219,6 +223,32 @@ export interface ZendeskConfig {}
 
 export interface DropboxConfig {}
 
+export type BlobType = "S3" | "R2" | "GOOGLE_CLOUD_STORAGE" | "OCI_STORAGE";
```

Comment:
these should be lowercase now?

Reply:
Meant to delete that! Not used anywhere!